srcpkgs/dhewm3/template

```diff
@@ -1,14 +1,17 @@
 # Template file for 'dhewm3'
 pkgname=dhewm3
-version=1.4.1
-revision=2
+version=1.5.0
+revision=1
+build_wrksrc="neo"
 build_style=cmake
-makedepends="zlib-devel libjpeg-turbo-devel libogg-devel
- libvorbis-devel libopenal-devel SDL2-devel libcurl-devel"
-short_desc="A Doom 3 GPL source port"
+makedepends="SDL2-devel libcurl-devel libjpeg-turbo-devel libopenal-devel"
+short_desc="Doom 3 GPL source port"
 maintainer="John Rowley <enterthevoid@codesector.co>"
-license="GPL-3"
-homepage="https://github.com/dhewm"
-distfiles="${homepage}/${pkgname}/archive/${version}.tar.gz"
-checksum=587586142d863ff98051619f0d49c056c5302e919c655a50af69a8293bc3ae08
-build_wrksrc="neo"
+license="GPL-3.0-or-later"
+homepage="https://dhewm3.org/"
+distfiles="https://github.com/dhewm/dhewm3/archive/${version}.tar.gz"
+checksum=fd299c8a2db1ecde15f861e4a883a5d61e5da0c74e26f89fb50b52b978fe8212
+
+case "$XBPS_TARGET_MACHINE" in
+	arm*) makedepends+=" libX11-devel";;
```

Comment:
Why is this needed? I have no clue.

Reply:
Take it out and see if arm cross still works, if it works then remove it.